### ISSUE TITLE
Any plans to support bridge network on Apple silicon

### ISSUE THREAD

**Issue by JCzz** [canonical/multipass#2424]
Hi

I would love to use Multipass, I am just waiting on being able to bridge to network adapter to give each server an IP address.

I get this on apple silicon

```
multipass networks
networks failed: The networks feature is not implemented on this backend.
```

**Comment by Saviq** [comment]
Hi @JCzz, yes, we'll be closing that gap with other drivers soon.

**Comment by txthinking** [comment]
If this, we can create Transparent Proxy  on macos as router gate way via TPROXY mod by https://txthinking.github.io/brook/#/brook-tproxy

**Comment by JCzz** [comment]
Thanks @txthinking 

I am betting on this to be included in multipass.

Hi @Saviq 
Do you have any information on progress on this issue, with all respect to the work on this? 
I am looking forward.

**Comment by Saviq** [comment]
@JCzz in progress here: #2462, will be included in our next release.

**Comment by Saviq** [comment]
Duplicate of #2364
